crates/bevy_render/src/lib.rs

```diff
@@ -149,6 +160,18 @@ impl Render {
                 .chain(),
         );
 
+        schedule.configure_sets((ExtractCommands, PrepareAssets).chain());
```

Comment:
```suggestion
       schedule.configure_sets((ExtractCommands, PrepareAssets, Prepare).chain());
```
it's not strictly necessary at the moment, but i think it would be useful to add an extra constraint to force `PrepareAssets` to finish before `Prepare`. Lots of the `Prepare` systems use `RenderAssets<Image>` and this will avoid breaking if we remove the need for images from earlier systems. Also without this there's no constraint from `PrepareAssets` to `Render` so if people use custom renderassets for the first time in the rendergraph it could theoretically blow up.

Reply:
I totally agree here, looking at the render graph the main oddity I noticed was the lack of constraints on PrepareAssets relative to the rest of the graph.